packages/nx/src/migrations/update-15-0-0/prefix-outputs.spec.ts

```diff
@@ -83,6 +83,10 @@ describe('15.0.0 migration (prefix-outputs)', () => {
   });
 
   it('should migrate package.json projects', async () => {
+    updateJson(tree, 'package.json', (json) => {
+      json.workspaces = ['proj'];
+      return json;
+    });
```

Comment:
Generators used to detect all **_new_** package.json's as a project immediately. This was incorrect behavior and is addressed in the changes to project-configuration.ts